build.rs

```diff
@@ -74,17 +74,21 @@ fn handle_kernel_version_cfg(bindings_path: &PathBuf) {
 }
 
 fn main() {
-    println!("rerun-if-env-changed=KDIR");
-    let output = String::from_utf8(
-        Command::new("make")
-            .arg("-C")
-            .arg("kernel-cflags-finder")
-            .arg("-s")
-            .output()
-            .unwrap()
-            .stdout,
-    )
-    .unwrap();
+    println!("cargo:rerun-if-env-changed=KDIR");
+    println!("cargo:rerun-if-env-changed=CLANG");
+    println!("cargo:rerun-if-changed=kernel-cflags-finder/Makefile");
+    let output = Command::new("make")
+        .arg("-C")
+        .arg("kernel-cflags-finder")
+        .arg("-s")
+        .output()
+        .unwrap();
+    if !output.status.success() {
+        eprintln!("kernel-cflags-finder did not succeed");
+        eprintln!("stdout: {}", std::str::from_utf8(&output.stdout).unwrap());
```

Comment:
Ooops, `std` prefix here is also not necessary.

Reply:
Fuck. I had it completely backwards, the `std::` prefix is in fact _entirely_ necessary. These are module functions, not assosciated functions of the `str` type.